pkg/k8s/synced/crd.go

```diff
@@ -244,6 +244,8 @@ var (
 		crdResourceName(v2.CNName),
 		crdResourceName(v2.CIDName),
 		crdResourceName(v2.CLRPName),
+		crdResourceName(v2.CCLRPName),
```

Comment:
Why is CCLRP also being added?

Reply:
I believe it is also being removed in a pending PR: https://github.com/cilium/cilium/pull/13671/commits/33e8399a806fe217606721ca87118819ef16a20b

Reply:
Only saw it was missing, was not aware of the PR for removing it. Will remove :-)